src/main/java/edu/harvard/iq/dataverse/DataCitation.java

```diff
@@ -619,8 +621,7 @@ private void createEndNoteXML(XMLStreamWriter xmlw) throws XMLStreamException {
         }
         if (persistentId != null) {
             xmlw.writeStartElement("electronic-resource-num");
-            String electResourceNum = persistentId.getProtocol() + "/" + persistentId.getAuthority() + "/"
-                    + persistentId.getIdentifier();
+            String electResourceNum = persistentId.asString();
```

Comment:
Has it been verified that an initial : after protocol is OK for EndNote import? It seems like the original code used / because ICPSR did. Can someone look at what EndNote exports and make sure we match it here? @adam3smith says he can check this next week if no one else has access.

Reply:
I see, I wondered about that slash there. I tried to find some info on the EndNote XML format but found only [this schema](https://support.clarivate.com/Endnote/s/article/EndNote-XML-Document-Type-Definition?language=en_US), which didn't specify anything about the format of `electronic-resource-num`, so I thought it might have been a mistake to use the slash here. It would be great if someone could check.